src/main/resources/transports/transports.tsv

```diff
@@ -4547,3 +4547,19 @@
 # Lletya Entrance								
 2304 3194 0	2306 3194 0	Pass Tree 8742					517>1	5
 2306 3194 0	2304 3194 0	Pass Tree 8742					517>1	5
+								
+# Mountain Guide (Varrock Museum)
+1401 3536 0	1275 3559 0	Mount Quidamortem 7600						7
```

Comment:
These are locked behind varbit 5421 (value should be 1 -- 5421=1)
MenuOption should be 'Travel' and MenuTarget should be 'Mountain Guide'

I think these don't belong here, since they're NPC related not ObjectID
Maybe the best solution would be to add a new transport TSV file for NPC related transports

That way the plugin could highlight a NPC an additional show a text above the plugin with the destination you should take (as seen with other transports now as well, for example quentzals)

Reply:
The 5421 varbit seems to be correct as pointed out by Owain, because the `Talk-to Mountain Guide 7599` turns into `Travel Mountain Guide 7600`, as can be seen here: https://chisel.weirdgloop.org/moid/npc_id.html#7599+1

Reply:
Additionally the mountain guide wouldn't be present at the locations other than Mount Quidamortem if you haven't talked to him first (which sets the varbit).